src/components/modal/walletModal.tsx

```diff
@@ -0,0 +1,156 @@
+import styled, { css } from 'styled-components';
+import { Modal } from './modal';
+import { fontSize, mediaQuery, pxToRem, spacing } from '../../styles';
+import { Button } from '../button/button';
+import { JazzIcon } from '../wallet/wallet';
+import { useState } from 'react';
+import { ButtonEnum, buttonType, useStore } from '../../helpers';
+import { useEthers } from '@usedapp/core';
+
+type Props = {
+	showModal: boolean;
+	setShowModal: (showModal: boolean) => void;
+	account: string;
+	isCopied?: boolean;
+};
+
+const ModalWrapper = styled.div`
+	display: flex;
+	align-items: center;
+	padding: ${spacing[18]} ${spacing[46]} 0 ${spacing[26]};
+
+	${mediaQuery('xs')} {
+		flex-direction: column;
+		align-items: flex-start;
+		padding: ${spacing[18]} ${spacing[46]} 0 ${spacing[14]};
+		margin-bottom: ${spacing[8]};
+	}
+`;
+
+const ModalContainer = styled.div`
```

Comment:
You can get rid of the `ModalContainer` by adding a `gap` attribute to the parent

Reply:
I need this as div to make flex style in modal 
Maybe i did not get what did you mean here, sorry.